python/ray/train/tensorflow/tensorflow_checkpoint.py

```diff
@@ -48,9 +70,179 @@ def from_model(
         checkpoint = cls.from_dict(
             {PREPROCESSOR_KEY: preprocessor, MODEL_KEY: model.get_weights()}
         )
+        checkpoint._flavor = cls.Flavor.MODEL_WEIGHTS
+        return checkpoint
+
+    @classmethod
+    def from_h5(
+        cls, file_path: str, *, preprocessor: Optional["Preprocessor"] = None
+    ) -> "TensorflowCheckpoint":
+        """Create a :py:class:`~ray.air.checkpoint.Checkpoint` that stores a Keras
+        model - from SavedModel format.
+
+        The path must maintain valid even after this function returns.
+
+        Args:
+            file_path: The same path as used by ``model.save(path)``.
+            preprocessor: A fitted preprocessor to be applied before inference.
+
+        Returns:
+            A :py:class:`TensorflowCheckpoint` converted from h5 format.
+
+        Examples:
+
+        .. code-block: python
```

Comment:
Please make sure to use a format that actually runs this in the CI, see my PR https://github.com/ray-project/ray/pull/28421 -- and also communicate it to the rest of the team :)

Reply:
Ah I see. Thanks for linking it to me :)  Just or my understanding, is that we should always use 

```
>>>
```
format instead of what is being done here for code to run in the CI?

Reply:
Yes, and then you should check in the `make doctest` part of the ci (it is called `Documentation` and has LINT=1) that the tests actually run. Maybe put an `assert False` in there and check it fails to make sure it actually ran.